en/what-do-i-need-to-publish.rst

```diff
@@ -0,0 +1,31 @@
+What do I need to publish?
+=========
+
+Publishing to IATI means you are telling a story about the work your organisation does. For this you break your work down into activities, these could be your programmes or projects. For each activity you detail what the work is, where and when it is happening, who is involved and how it is being financed etc. There are loads of details you can add for each activity. The IATI technical team recommends that you start by adding the details (in the form of elements) below:
```

Comment:
`loads` [of details] -> `many` or `lots` or something.

(`loads` is a bit colloquial and doesn't translate cleanly)